runelite-client/src/test/java/net/runelite/client/plugins/grounditems/WildcardMatchLoaderTest.java

```diff
@@ -34,12 +34,14 @@
 	@Test
 	public void testLoad()
 	{
-		WildcardMatchLoader loader = new WildcardMatchLoader(Arrays.asList("rune*", "Abyssal whip"));
-		assertTrue(loader.load("rune pouch"));
-		assertTrue(loader.load("Rune pouch"));
-		assertFalse(loader.load("Adamant dagger"));
-		assertTrue(loader.load("Runeite Ore"));
-		assertTrue(loader.load("Abyssal whip"));
-		assertFalse(loader.load("Abyssal dagger"));
+		WildcardMatchLoader loader = new WildcardMatchLoader(Arrays.asList("rune* <5", "Abyssal whip", "nature* > 20"));
```

Comment:
Rather than modifying this test, add another test case which tests against quantity matchers.